src/Raven.Server/Documents/Indexes/Persistence/Corax/LuceneAnalyzerAdapter.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using Corax;
+using Corax.Pipeline;
+using Lucene.Net.Analysis.Tokenattributes;
+using LuceneAnalyzer = Lucene.Net.Analysis.Analyzer;
+
+namespace Raven.Server.Documents.Indexes.Persistence.Corax
+{
+    public sealed unsafe class LuceneAnalyzerAdapter : Analyzer
```

Comment:
why do we want this?

Reply:
Because we want to be able to use the Lucene analyzers too. This will adapt the Lucene adapter pipeline to the Corax Pipeline and allow the users to use older Lucene analyzers they already have implemented.

Reply:
Why do we want *that*? It would be easier for us to not take that on ourselves.
Re-writing an analyzer is an acceptable cost for moving to Corax.

Reply:
It helps us run all the tests that we have. So there is also an internal reliability story here.